### DISCUSSION TITLE
Trim URL of ParentLink property

### DISCUSSION THREAD

**Discussion by xomi-i** [microsoft-search/pnp-modern-search#4086] [Q&A]
Hello,

I am currently doing a project in SharePoint that uses PnP Search Box Result. I want to customize the column using handlebars where I have to trim the URL of the ParentLink, here's a sample:

parentFolder/jobFolder/customerFolder/Assign Docs/Evidence
Would become
parentFolder/jobFolder/customerFolder/Assign Docs

URL should be until "/Assign Docs" only

Here's my code:

![image](https://github.com/user-attachments/assets/1d8075e1-5a9a-4c32-8e95-e0c58f8092a8)

This code is returning this kind of error:
![image](https://github.com/user-attachments/assets/5911cdba-de56-46f7-8c39-641981c213ba)

Error: Missing helper: "trimFolderPath"

May I know if there is someone out there that encountered this kind of error as well? May I know what you did to resolve this?

Thanks!

**Comment by kasperbolarsen** [comment]
If you need a custom helper please take a look at https://microsoft-search.github.io/pnp-modern-search/extensibility/handlebars_customizations/

**Reply by wobba** [reply]
What Kasper said as you cannot add javascript to the templates.
